Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -187,15 +202,21 @@ class HomeFragmentViewModel @AssistedInject constructor(
             if (statsData.isDataAvailable) {
                 add(
                     StatisticsHomeCard.Item(
-                        // TODO: improve in future PRs (EXPOSUREAPP-7446) isEnable depends on number of Local Cards
-                        data = statsData.copy(items = mutableListOf(AddStatsItem(true)).plus(statsData.items)),
+                        data = statsData,
                         onClickListener = {
                             when (it) {
                                 is AddStatsItem -> {
                                     events.postValue(HomeFragmentEvents.GoToFederalStateSelection)
                                 }
                                 else -> events.postValue(HomeFragmentEvents.GoToStatisticsExplanation)
                             }
+                        },
+                        onRemoveListener = {
+                            localStatisticsConfigStorage.activeDistricts.update { districts ->
+                                districts.filter { selected ->
+                                    selected.district.districtId != it.selectedDistrict?.district?.districtId
+                                }.toSet()
+                            }
```

Comment:
Something with the removal doesn't seem to work as intended.
After playing around I end up with this (duplicates and contains already removed items):

![Screenshot from 2021-07-12 17-44-47](https://user-images.githubusercontent.com/1439229/125317070-00250800-e339-11eb-8368-3ba40cc25214.png)

Reply:
i've removed the nullability, maybe that already fixes it but if not i will investigate further